bedrock/base/templates/includes/global-nav.html

```diff
@@ -0,0 +1,277 @@
+<nav role="navigation" class="moz-global-nav" id="moz-global-nav">
+    <div class="nav-horizontal-menu">
+
+      <button type="button" class="nav-button-menu nav-hidden" id="nav-button-menu" aria-controls="moz-global-nav-drawer">
+        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 25" width="12" height="25">
+          <title>{{_('Menu')}}</title>
+            <rect class="p1" fill="#231f20" x="3" width="5" height="6"/>
+            <rect class="p2" fill="#231f20" x="3" y="9.5" width="5" height="6"/>
+            <rect class="p3" fill="#231f20" x="3" y="19" width="5" height="6"/>
+        </svg>
+      </button>
+
+      <h2 class="nav-logo">
+        <a href="{{ url('mozorg.home') }}">
+          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 578.55 185.54" width="78" height="25">
+            <title>{{_('Mozilla')}}</title>
+            <path class="p" d="M503.5 117.21c0 4.92 2.37 8.82 9 8.82 7.8 0 16.11-5.6 16.61-18.31a80.86 80.86 0 0 0-11-1c-7.83-.01-14.61 2.19-14.61 10.49z"/>
+            <path class="p" d="M0 0v185.54h578.55V0zm163.78 139.93h-32V96.87c0-13.22-4.41-18.31-13.05-18.31-10.51 0-14.75 7.46-14.75 18.14v26.64h10.12v16.61h-32V96.87c0-13.22-4.4-18.31-13.05-18.31-10.51 0-14.75 7.46-14.75 18.14v26.64h14.54v16.61H22.22v-16.61h10.17V80.09h-11V63.48h32.87V75c4.58-8.13 12.55-13.05 23.22-13.05 11 0 21.19 5.26 24.92 16.45 4.24-10.17 12.88-16.45 24.92-16.45 13.73 0 26.28 8.31 26.28 26.45v34.94h10.17zm48.65 1.69c-23.56 0-39.84-14.41-39.84-38.82 0-22.38 13.56-40.86 41-40.86s40.86 18.48 40.86 39.84c.02 24.42-17.61 39.85-42.02 39.85zm121.72-1.69h-66.8l-2.2-11.53 42-48.32h-23.9l-3.39 11.87-15.77-1.69 2.71-26.79H334L335.69 75l-42.4 48.34H318l3.56-11.87 17.29 1.69zm41.36 0h-22.89v-27.46h22.89zm0-49h-22.89V63.48h22.89zm12 49L420.6 23.34h21.53l-33.06 116.59zm44.42 0L465 23.34h21.53l-33.04 116.59zm113.92 1.69c-10.17 0-15.76-5.94-16.78-15.26-4.41 7.8-12.21 15.26-24.58 15.26-11 0-23.56-5.94-23.56-21.87 0-18.82 18.14-23.22 35.6-23.22a100.23 100.23 0 0 1 12.55.68v-2.54c0-7.8-.17-17.12-12.55-17.12-4.58 0-8.14.34-11.7 2.2L502 90.6l-17.46-1.87 3.39-19.83c13.39-5.43 20.17-7 32.72-7 16.45 0 30.35 8.48 30.35 25.94v33.23c0 4.41 1.69 5.94 5.26 5.94a11.5 11.5 0 0 0 3.22-.51l.17 11.53a29.57 29.57 0 0 1-13.77 3.6z"/>
+            <path class="p" d="M213.27 78.73c-11.19 0-18.14 8.3-18.14 22.72 0 13.22 6.1 23.39 18 23.39 11.36 0 18.82-9.15 18.82-23.73-.03-15.43-8.33-22.38-18.68-22.38z"/>
+          </svg>
+        </a>
+      </h2>
+
+      <ul class="nav-primary-links">
+        <li class="item-orange"><a href="{{ url('firefox.new') }}" data-id="firefox">{{ _('Firefox') }}</a></li>
```

Comment:
I'm not sure how I feel about these presentational classes. I know that at least for now they really are presentational and are only being used as hooks for colors, but maybe it would be better to classify these items based on the section/category of their contents, then define a theme color for each one in the CSS.